datadog_checks_dev/datadog_checks/dev/tooling/commands/validate/config.py

```diff
@@ -35,9 +38,9 @@ def config(check):
             num_files += 1
             file_display_queue = []
             file_name = basepath(config_file)
-
+            file_data = read_file(config_file)
```

Comment:
can read_file throw an exception?

Reply:
Yes, this should have stayed in the try block